funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -75,23 +67,16 @@ class EndpointInterchange:
     5. Be aware of requests worker resource capacity,
        eg. schedule only jobs that fit into walltime.
 
-    TODO: We most likely need a PUB channel to send out global commandzs, like shutdown
     """
 
     def __init__(
         self,
-        config,
-        client_address="127.0.0.1",
-        interchange_address="127.0.0.1",
-        client_ports: Tuple[int, int, int] = (50055, 50056, 50057),
-        launch_cmd=None,
+        config: funcx_endpoint.endpoint.utils.config.Config,
+        reg_info: pika.connection.Parameters = None,
         logdir=".",
         endpoint_id=None,
-        keys_dir=".curve",
-        suppress_failure=True,
```

Comment:
Is `suppress_failure` related? It's not clear if that refers to ZMQ-specific errors or what.